auth0/structure_auth0_connection.go

```diff
@@ -845,3 +855,20 @@ func castToListOfStrings(interfaces []interface{}) *[]string {
 	}
 	return &strings
 }
+
+func expandSAMLSigningKey(d ResourceData) *management.ConnectionOptionsSAMLSigningKey {
+	var expandedSigningKey management.ConnectionOptionsSAMLSigningKey
+
+	signingKey := List(d, "signing_key")
+
+	if signingKey.Len() == 0 {
+		return nil
+	}
+
+	signingKey.Elem(func(d ResourceData) {
+		expandedSigningKey.Cert = String(d, "cert")
+		expandedSigningKey.Key = String(d, "key")
+	})
+
+	return &expandedSigningKey
+}
```

Comment:
We don't need this check:

```go
if signingKey.Len() == 0 {
		return nil
}
```

Because the `Elem()` func will skip the loop if signingKey has a length of 0. 

This means we can rewrite the whole func as follows:

```go
func expandSAMLSigningKey(d ResourceData) *management.ConnectionOptionsSAMLSigningKey {
	var expandedSigningKey *management.ConnectionOptionsSAMLSigningKey

	List(d, "signing_key").Elem(func(d ResourceData) {
		expandedSigningKey = &management.ConnectionOptionsSAMLSigningKey{
			Cert:  String(d, "cert"),
			Key: String(d, "key"),
		}
	})

	return expandedSigningKey
}

```